modules/gdscript/editor/script_templates/CharacterBody2D/basic_movement.gd

```diff
@@ -5,26 +5,26 @@ extends _BASE_
 
 const SPEED = 300.0
 const JUMP_VELOCITY = -400.0
+const FRICTION = 600.0
 
-# Get the gravity from the project settings to be synced with RigidBody nodes.
-var gravity: int = ProjectSettings.get_setting("physics/2d/default_gravity")
+# Get the gravity from the project settings, to allow syncing with RigidBody2D nodes.
+var gravity = ProjectSettings.get_setting("physics/2d/default_gravity")
 
 
-func _physics_process(delta: float) -> void:
+func _physics_process(delta):
```

Comment:
The type hints will be automatically stripped from the template when creating the script, they are supposed to be included in the template so that they will exist when users enable type hints in editor settings.

Reply:
I see - will fix.